box-ui/src/components/ts/ConnectionComponent.tsx

```diff
@@ -8,9 +8,11 @@ import MarshaLoginComponent from './MarshaLoginComponent';
 import HighlightOffIcon from '@mui/icons-material/HighlightOff';
 import { Button } from '@mui/material';
 import ArrowBackRoundedIcon from '@mui/icons-material/ArrowBackRounded';
-import { ConnectionProps } from '../../types';
+import { ConnectionProps } from '../../utils/Props';
```

Comment:
You may have failed your rebase, IIRC this was already done in PR #34 (as it can be seen here: https://github.com/openfun/jitsi-box/blob/develop/box-ui/src/components/ts/ConnectionComponent.tsx)
I think it is the same issue in many components, and particularly in JitsiComponent.tsx